src/tools.sh

```diff
@@ -1,6 +1,9 @@
 #!/usr/bin/env bash
-gnomeVersion="$(expr "$(gnome-terminal --version)" : '.* \(.*[.].*[.].*\)$')"
+#gnomeVersion="$(expr "$(gnome-terminal --version)" : '.* \(.*[.].*[.].*\)$')"
```

Comment:
Don't comment the old version of the line you replaced. Again, git already handles the history.